tls/s2n_config.c

```diff
@@ -249,8 +249,11 @@ int s2n_config_defaults_init(void)
 
 void s2n_wipe_static_configs(void)
 {
-    s2n_config_cleanup(&s2n_default_config);
-    s2n_config_cleanup(&s2n_default_fips_config);
+    if (s2n_is_in_fips_mode()) {
+        s2n_config_cleanup(&s2n_default_fips_config);
+    } else {
+        s2n_config_cleanup(&s2n_default_config);
+    }
```

Comment:
This seems prone to future bugs if we change our logic for when to create the various configs. Can we instead just make s2n_config_cleanup handle nulls properly?